cpu/cortexm_common/include/cpu_conf_common.h

```diff
@@ -107,7 +107,7 @@ extern "C" {
  * If you want to set this, define it in your `cpu_conf.h`.
  */
 #ifndef CPU_CORTEXM_PRIORITY_GROUPING
-#define CPU_CORTEXM_PRIORITY_GROUPING   (0)
+#define CPU_CORTEXM_PRIORITY_GROUPING   (1)
```

Comment:
The above comment says

>  A currently experimental way to make PendSV on Cortex-M to run last is to
>  - make sure @ref CPU_DEFAULT_IRQ_PRIO is even (e.g. `6U`),
>  - set @ref CPU_CORTEXM_PENDSV_IRQ_PRIO to one higher, and
>  - set this one

But when checking, most implementations will set

```C
#define CPU_DEFAULT_IRQ_PRIO            (1U)
```

in `cpu_conf.h`, violating this advice.

(Which begs the question: Why not also move the definition of `CPU_DEFAULT_IRQ_PRIO` to `cpu_conf_common.h`?)